src/main/java/duke/util/Parser.java

```diff
@@ -0,0 +1,98 @@
+package duke.util;
+
+import duke.Apollo;
+import duke.tasks.Task;
+import duke.tasks.TaskHandler;
+import duke.ui.Ui;
+import duke.enums.Commands;
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidOptionException;
+import duke.exceptions.UnrecognisedCommandException;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+/**
+ * Represents a parser that handles and makes sense of user input.
+ */
+public class Parser {
+
+    /**
+     * Handles and makes sense of input provided by the user.
```

Comment:
Would "Executes relevant command instructions based on user input" be better here? "Handles and makes sense" sounds a bit vague when I first read it :sweat_smile: